test/infrastructure/host-cluster.sh

```diff
@@ -0,0 +1,201 @@
+# Environment Variables Config
+export CLUSTER_NAME=testing-ci-cluster-2
+export GUEST_CLUSTER_NAME=example-guest-cluster
+export AWS_PROFILE=karpenter-ci
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+export AWS_REGION=us-west-2
+export KARPENTER_VERSION=v0.10.1
+
+# Create Cluster
+cat <<EOF > cluster.yaml
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: ${CLUSTER_NAME}
+  region: ${AWS_REGION}
+  version: "1.21"
+  tags:
+    karpenter.sh/discovery: ${CLUSTER_NAME}
+managedNodeGroups:
+  - instanceType: m5.large
+    amiFamily: AmazonLinux2
+    name: ${CLUSTER_NAME}-ng
+    desiredCapacity: 1
+    minSize: 1
+    maxSize: 10
+iam:
+  withOIDC: true
+EOF
+eksctl create cluster -f cluster.yaml
+#----------------
+# Sleep 10 in between each to allow Tekton CRDs to install.
+kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.33.2/release.yaml
+sleep 10
+kubectl apply -f https://storage.googleapis.com/tekton-releases/triggers/previous/v0.19.0/release.yaml
```

Comment:
do these depend on each other to be completely up or should they reconcile to a good state if you apply them all at once?

Reply:
There's odd behavior with Tekton if you apply these all too close to each other. Sleeping for 10 seconds makes it enough seconds. When we automate this with something like Flux, this shouldn't be a problem.